### PR TITLE
Fix controller timing issue

### PR DESCRIPTION
**Pull request by YuanTingHsieh**
### Description

The problem is related to this part of codes: https://github.com/NVIDIA/NVFlare/blob/dev/nvflare/apis/impl/seq_relay_manager.py#L112-L120 

The old controller code has a rare chance that a `client_task` has already been put inside the `task.last_client_task_map[client.name]` BUT its `task_sent_time` is None, which will cause this following line to failed:

`if task_result_timeout and time.time() - last_task.task_sent_time > task_result_timeout:`

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [x] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: nvflare/apis/impl/controller.py [hunk 1]
```diff
@@ -181,12 +181,6 @@ def process_task_request(self, client: Client, fl_ctx: FLContext) -> Tuple[str,
 
                 if client_task_to_check is not None:
                     # this client has been sent the task already
```
[the remark this answers is not in the document]
Comment (YuanTingHsieh) [reply nvflare/apis/impl/controller.py:183]: yes we still need it

**Review by yanchengnv — DISMISSED** [review]
LGTM

**Pull request merged**